wrappers/services.go

```diff
@@ -101,7 +92,7 @@ CPUAccounting=true
 	if grp.CPULimit != nil && grp.CPULimit.Percentage != 0 {
 		// convert the number of cores and the allowed percentage
 		// to the systemd specific format.
-		cpuQuotaSnap := max(grp.CPULimit.Count, 1) * grp.CPULimit.Percentage
+		cpuQuotaSnap := grp.GetCorrectedCPUCount() * grp.CPULimit.Percentage
```

Comment:
Hmm what if I boot with n CPUs, set the quota, and then boot again with a different CPU count? In this case the quota would be incorrect, wouldn't it?

Reply:
That depends entirely on how the quota was set, if you set the quota as --cpu 2x50%, then the quota will be identical, if you have set the quota as --cpu 50%, then it would automatically scale to the new number of CPUs

Reply:
But if we have less CPUs after reboot (and a quota previously set with concrete number), it won't be correct? Not sure if monitoring for cpu count changes is within the scope of this feature, maybe a material for discussion and a followup?

Reply:
Ah now I understand, so yes, if you set a concrete number (e.g 2x50%), and you actually have 1 core, then it'll translate to 1x50% instead of 2x50% - yea I agree this might not be correct. The value set by GetCorrectedCPUCount should always use the set number of cores (if one is provided) no matter if it is higher than the actual available number of cores, that would make sense to me.

Reply:
I have changed this now, so 2x50% correctly expands to 100%, no matter the cores of on the system.